pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "niceml"
-version = "0.12.0"
+version = "0.12.1"
```

Comment:
Please do not update the niceML version manually. This will be done automatically by the release pipeline.

Reply:
Reverted the change